guides/common/modules/proc_adding-rhv-connection.adoc

```diff
@@ -1,5 +1,5 @@
 [id="adding-rhv-connection_{context}"]
-= Adding the {oVirt} Connection to {ProjectServer}
+= Adding the {oVirt} connection to {ProjectServer}
```

Comment:
same as above but slightly more tricky. an/a oVirt/RHV connection

Reply:
I agree but this does not have to be fixed in this PR, especially since oVirt support is likely to be dropped in 6.15.